templates/emails/base.html

```diff
@@ -482,7 +482,7 @@
                     {% block footer %}
                     {% endblock %}
                   </p>
-
+                  <p class="f-fallback sub align-center" style="font-size: 13px; line-height: 1.625; text-align: center; color: #A8AAAF; margin: .4em 0 1.1875em;" align="center">Do you have a question? Contact us at <a href="https://app.simplelogin.io/dashboard/support">https://app.simplelogin.io/dashboard/support</a></p>
```

Comment:
I think we can just keep "Contact us" in the href link. So something like `<a href="https://app.simplelogin.io/dashboard/support">A question?</a>`

Reply:
Changed.